### ISSUE TITLE
Simplified

### ISSUE THREAD

**Issue by Mercurian** [ekrividus/autoMB#11]
Is it possible to slim this down dramatically to just 

If light skillchain, cast thunder 5

This is way more complex than I think it needs to be.

**Comment by ekrividus** [comment]
This wasn't designed for a single job to single burst. It was designed to support every functional MBing job and the options for different spells and scenarios. A BLM that has SCH support really wants to capitalize on weather effects provided and isn't going to want to just MB a single T5 spell for instance. A nin isn't going to be using regular elemental magic at all. A SCH may want to use Helix spells. I wanted to support all of that.

If you want it to only single burst and just use a T5. Out of the box without any settings changes that should happen. Just load it up when you're on BLM. If you want it to only ever (on every single job) perform that way. After it's loaded you only have to ```//amb c spell``` with all other settings left to default. It'll just single burst the best spell for the skillchain.

**Comment by Mercurian** [comment]
The thing is it works exactly like that, except on the second burst. I exclusively want thunder, not wind, not fire. 

And on darkness I only want ice. I don't want it checking day or weather or alternate elements. How much can I delete to make that happen.

**Comment by ekrividus** [comment]
```//autoMB sd``` a couple of times until step down is on always. Then it will use the same element one tier down instead of cycling elements. Unless you have sub 2 second cooldowns on T5 spells, then it might use T5>T5 instead of T5>T4.

If you want to go in and mess with the logic to try to make it behave by only MBing on Light or Dark and then only with Thunder or Blizzard you're more than welcome to. It will take more than just deleting a few lines though. Honestly to do that you'd probably be better off pulling out the parts that detect a Skillchain and write up new spell selection logic from scratch.
But since a couple of changes to it's settings can make it do what you're asking for that seems unnecessary unless you just want to learn/practice :)

**Comment by Mercurian** [comment]
Somewhere in the code thunder 3 only comes out as /ma III

**Comment by ekrividus** [comment]
That's on the list of things I need to look into if/when I return to the game.

**Comment by Mercurian** [comment]
How about switching the spell order so it's thunder fire wind, can I do that?

**Comment by ekrividus** [comment]
Yeah that's fairly easy. Around line 125 you'll find a spell priorities table. Just change the ordering of the spells in there to whatever floats your boat.

**Comment by Mercurian** [comment]
I did exactly that, and it still casts thunder wind fire

**Comment by Mercurian** [comment]
lol even removing aero entirely from the priority list doesn't work. the hell.

**Comment by Mercurian** [comment]
Aha. I got it now. Change line 80.
local skillchains = T{
	[288] = {id=288,english='Light',elements={'Thunder','Fire','Wind'}},
